renku_notebooks/api/amalthea_patches/git_proxy.py

```diff
@@ -20,6 +20,12 @@ def main(server):
                     "path": "/statefulset/spec/template/spec/containers/-",
                     "value": {
                         "image": current_app.config["GIT_HTTPS_PROXY_IMAGE"],
+                        "securityContext": {
+                            "fsGroup": 100,
+                            "runAsGroup": 100,
```

Comment:
1000?

Reply:
The jupyterlab images run with fsgroup/group 100. Let me see if we can switch fully to 1000.

Reply:
It works.